client/gutenberg/editor/style.scss

```diff
@@ -69,7 +69,7 @@ $gutenberg-theme-toggle: #11a0d2;
 	display: none;
 }
 
-.is-section-gutenberg-editor {
+.layout.is-section-gutenberg-editor {
```

Comment:
Not sure what fixes these changes in the styles

Reply:
`.is-section-gutenberg-editor` is added to both the `body` and the `div.layout` element, which is the closest wrapper of the actual section.

If we apply the Gutenberg style to `body.is-section...`, we also end up applying it to the Calypso modals, who don't really need it.
AFAIK, you can notice that the Video and Audio tabs for Free sites look off (the heading text lacks the margin, the banner icon is too small).

Applying the Gutenberg style to `.layout.is-section...` instead, makes sure we only apply it to Gutenberg.